src/api/erc20/Erc20ApiMock.ts

```diff
@@ -36,6 +36,21 @@ export class Erc20ApiMock implements Erc20Api {
     return balance ? balance : ZERO
   }
 
+  public async name({ tokenAddress }: { tokenAddress: string }): Promise<string> {
+    log("Don't care about %s, just making TS shut up", tokenAddress)
```

Comment:
this is good enough, I think it's not bad at some point to add a mocked OWL, that is not in our list, but still return for his address the OWL symbol, name, and it's decimals.

If you want to try to create already some mock trades with OWL, you could see already if your service layer is resolving using the Erc20Api when it's not in our the list

Reply:
Yeah. I was not even going to implement it to get the PR out for feedback first, but TS was bugging me about it.
I intend to go back and add some mock functionality to it.

TODO

Reply:
Added the mock list of tokens to be `found`. Will add mock orders in a follow up PR